nuggetsapp/urls.py

```diff
@@ -1,7 +1,10 @@
-from django.conf.urls import url
+from django.conf.urls import url, include
 from . import views
 
 urlpatterns = [
+    url(r'^api/v1/nuggets/(?P<user_id>[0-9]+)/$', views.nuggets_for_user, name='nuggets_for_user'),
```

Comment:
This is a good resource: http://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api#requirements